.config/setup-scripts/setup-dotnet.sh##os.WSL,e.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+echo "Starting .NET setup for Linux..."
+
+# Install .NET SDK for Ubuntu/Debian
+echo "Installing .NET SDK..."
+curl -fsSL https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
```

Comment:
I think we further simplify this:
The old logic already considers installation with native tools tools. Please remove this file here again and make sure the default implementation also works without brew.
We can then also remove the yadm marker again

Don’t miss to update the reference in setup-work